test/e2e/pods.go

```diff
@@ -693,7 +693,8 @@ var _ = Describe("Pods", func() {
 									Port: intstr.FromInt(8080),
 								},
 							},
-							InitialDelaySeconds: 30,
+							InitialDelaySeconds: 15,
+							TimeoutSeconds:      30,
```

Comment:
To avoid possibility of flakes essentially. I think we care less about the
timeout itself and more about the functionality of probing.
This test used to fail in 31 seconds prior to this PR. With this change, it
might fail in 45 seconds. Do you think the additional 14 seconds will cause
timeouts, given that this test can be run in parallel?

On Mon, Feb 8, 2016 at 1:23 PM, Yu-Ju Hong <notifications@github.com> wrote:

> In test/e2e/pods.go
> <https://github.com/kubernetes/kubernetes/pull/20732#discussion_r52219987>
> :
>
> > @@ -693,7 +693,8 @@ var _ = Describe("Pods", func() {
> >  									Port: intstr.FromInt(8080),
> >  								},
> >  							},
> > -							InitialDelaySeconds: 30,
> > +							InitialDelaySeconds: 15,
> > +							TimeoutSeconds:      30,
>
> Bumping from 1s (default) to 30s seems a little bit extreme. Is there any
> reason to go so far and will the test timeout be able to accommodate this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20732/files#r52219987>.
>